log/value_test.go

```diff
@@ -10,26 +10,26 @@ import (
 
 func TestValueBinding(t *testing.T) {
 	var output []interface{}
+
 	logger := log.Logger(log.LoggerFunc(func(keyvals ...interface{}) error {
 		output = keyvals
 		return nil
 	}))
-	logger = log.With(logger, "ts", log.Timestamp, "caller", log.Caller)
+	logger = log.With(logger, "ts", log.DefaultTimestamp, "caller", log.DefaultCaller)
 
-	time.Sleep(50 * time.Millisecond)
```

Comment:
Well, now you can mock the time function so no need to actually sleep!

On Friday, April 24, 2015, Chris Hines <notifications@github.com> wrote:

> In log/value_test.go
> <https://github.com/ChrisHines/gokit/pull/1#discussion_r29079367>:
>
> >
> > -	time.Sleep(50 * time.Millisecond)
>
> I will figure out a smaller value that still works after I merge the
> changes from this discussion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ChrisHines/gokit/pull/1/files#r29079367>.
>


-- 
Sent from Gmail Mobile